Purchases/Networking/RCHTTPClient.m

```diff
@@ -179,16 +186,18 @@ - (NSMutableURLRequest *)createRequestWithMethod:(NSString *)HTTPMethod
         NSError *JSONParseError;
         NSData *body = [NSJSONSerialization dataWithJSONObject:requestBody options:0 error:&JSONParseError];
         if (JSONParseError) {
-            RCLog(@"Error creating request JSON: %@", requestBody);
+            RCErrorLog(@"Error creating request JSON: %@", requestBody);
+            return nil;
```

Comment:
new behavior: if we can't create a JSON request, don't create the request at all.